docs/service-fabric/refactor-migrated-app.md

```diff
@@ -0,0 +1,155 @@
+
+> [!NOTE]
+> For more information about setting up monitoring and diagnostics for Service Fabric applications running in production, see [monitoring and diagnostics][monitoring-diagnostics] for Azure Service Fabric.
+
+For example, the following lines in *startup.cs* for each of the web front end services should be commented out:
+
+```csharp
+// This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
+public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerFactory loggerFactory)
+{
+    //loggerFactory.AddConsole(Configuration.GetSection("Logging"));
+    //loggerFactory.AddDebug();
+
+    app.UseMvc();
+}
+```
+
+> [!NOTE]
+> These lines may be conditionally excluded when Visual Studio is set to “release” when publishing.
+
+Finally, when Tailspin deploys the Tailspin application to production, they switch Visual Studio to **release** mode.
+
+## Deployment considerations
+
+The refactored Surveys application is composed of five stateless services and one stateful service, so cluster planning is limited to determining the correct VM size and number of nodes. In the *applicationmanifest.xml* file that describes the cluster, Tailspin sets the *InstanceCount* attribute of the *StatelessService* tag to -1 for each of the services. A value of -1 directs Service Fabric to create an instance of the service on each node in the cluster.
+
+> [!NOTE]
+> Stateful services require the additional step of planning the correct number of partitions and replicas for their data.
+
+Tailspin deploys the cluster using the Azure Portal. The Service Fabric Cluster resource type deploys all of the necessary infrastructure, including VM scale sets and a load balancer. The recommended VM sizes are displayed in the Azure portal during the provisioning process for the Service Fabric cluster. Note that because the VMs are deployed in a VM scale set, they can be both scaled up and out as user load increases.
+
+> [!NOTE]
+> As discussed earlier, in the migrated version of the Surveys application the two web front ends were self-hosted using ASP.Net Core and Kestrel as a web server. While the migrated version of the Survey application does not use a reverse proxy, it is strongly recommended to use a reverse proxy such as IIS, Nginx, or Apache. In the refactored Surveys application, the two web front ends are self-hosted using ASP.Net Core with WebListener as a web server so a reverse proxy is not necessary.
+
+## Next steps
+
+The Surveys application code is available on [GitHub][sample-code].
+
+If you are just getting started with [Azure Service Fabric][service-fabric], set up your development environment and download the latest[Azure SDK][azure-sdk] and the [Azure Service Fabric SDK][service-fabric-sdk]. The SDK includes the OneBox cluster manager so you can deploy and test the Surveys application locally with full F5 debugging.
```

Comment:
Please add a space between "latest" and the link to Azure SDK.

Reply:
Fixed.